_docs-sources/reference/modules/intro.md

```diff
@@ -3,8 +3,17 @@
 # Fortunately, this property only seems to affect # auto-generated sidebar items.
 # So we still _do_ display this page in the sidebar, but use a manual entry.
 sidebar_class_name: dont_show
+pagination_next: null
```

Comment:
This is to fix a bug that was reported where the "next" link from this page just went back to itself. In this case - we actually don't want any "next" link showing up at all.